src/docs/content/tutorials/quickstart/index.md

```diff
@@ -1,213 +1,274 @@
+                  .acceptHeader("application/json")
+                  .contentTypeHeader("application/json");
 
-```
-Do you want to run the simulation locally, on Gatling Enterprise, or just package it?
-Type the number corresponding to your choice and press enter
-[0] <Quit>
-[1] Run the Simulation locally
-[2] Run the Simulation on Gatling Enterprise Cloud
-[3] Package the Simulation for Gatling Enterprise
+ScenarioBuilder myFirstScenario = scenario("My First Scenario")
+                .exec(http("Request 1")
+                        .get("/computers/"))
+                .pause(2)
+                .exec(http("Request 2")
+                        .get("/computers/new"));
+
+//Add the setUp block: 
+{
+        setUp(
+                myFirstScenario.injectOpen(constantUsersPerSec(2).during(60))
+        ).protocols(httpProtocol);
+    }
+}
 ```
 
-Press 1 then enter.
+### Run the test locally
 
-You should then see a menu with the simulation examples:
+The open-source version of Gatling allows you to run tests locally, generating load from your computer. Running a new or modified test locally is often useful to ensure it works before launching it on Gatling Enterprise Cloud. Using the Maven plugin, you can launch your test with the following command in the project root directory: 
+
+```bash 
+# Maven plugin
+mvn gatling:test
+
+# Maven wrapper Mac OS and Linux
+./mvnw gatling:test
+
+# Maven wrapper Windows
+mvnw.cmd gatling:test
 
 ```
-Choose a simulation number:
-[0] computerdatabase.BasicSimulation
-[1] computerdatabase.advanced.AdvancedSimulationStep01
-[2] computerdatabase.advanced.AdvancedSimulationStep02
-[3] computerdatabase.advanced.AdvancedSimulationStep03
-[4] computerdatabase.advanced.AdvancedSimulationStep04
-[5] computerdatabase.advanced.AdvancedSimulationStep05
-```
+When the test has finished, there is an HTML link in the terminal that you can use to access the static report. 
 
-Press 0 then enter. Press enter next when gatling asks for a run description.
+### Run the test on Gatling Enterprise Cloud
 
-When the simulation is done, the console will display a link to the HTML reports.
+Gatling Enterprise Cloud is a feature-rich SaaS platform that is designed for teams and organizations to get the most out of load testing. With the trial account you created in the [Sign up for Gatling Enterprise Cloud (optional)](#sign-up-for-gatling-enterprise-cloud-optional) section, you can upload and run your test with advanced configuration, reporting, and collaboration features. 
+
+#### Create an API token
+1. Navigate to your Gatling Enterprise Cloud account. 
+2. Select _API Tokens_ from, the left-side menu.
+3. On the API Tokens page, click the **Create** button in the top-right corner.
+4. In the modale:
+  - enter a token name,
+  - select _configure_ for the organizational role,
+  - choose the team,
+  -  select _configure_ for the team role,
+  - click **Save**.
+5. Copy the generated token to your local machine. 
```

Comment:
A reminder that they won't be able to copy it anymore? (but they can regenerate a new one)